s3o_import.py

```diff
@@ -390,21 +398,36 @@ def new_material(tex1, tex2, texsdir, name="Material"):
         #load diffuse texture, plug in UV mapping, link to base color.
         fname = find_in_folder(texsdir, tex1)
         image = bpy.data.images.load(os.path.join(texsdir, fname))
+        image.alpha_mode = 'CHANNEL_PACKED' #spring uses alpha as teamcolor
         tex_node = mat.node_tree.nodes.new('ShaderNodeTexImage')
         tex_node.image = image
         mat.node_tree.links.new(principled.inputs['Base Color'], tex_node.outputs['Color'])
         mat.node_tree.links.new(tex_node.inputs['Vector'], mapping_node.outputs['Vector'])
         
     if tex2 and find_in_folder(texsdir, tex2):
-        #load specular texture, plug in same UV map, set to non colour data and link to specularity.
+        #load reflectivity / emission / data texture, plug in same UV map, set to non colour data and link to appropriate data.

```

Comment:
80 characters max please.
Send "to non colour data and link to appropriate data." to the next line